server/search.py

```diff
@@ -46,7 +51,7 @@ def _get_variants_in_range(self, chr, start, end):
         else:
             raise utils.NotFoundException(str(chr) + ':' + str(start) + '-' + str(end))
 
-    def _search_gene(self, query):
+    def _search_gene(self, query, data_type):
```

Comment:
in all of these searches, remove the exception where no results exist and return empty list as appropriate. an always in UI side tell user that.